### PR TITLE
Feature/14 welcome screen guest UI

### LINKED ISSUE #14: Welcome Screen Guest UI
# REQUIREMENTS
Develop the welcome screen (guest).
# DESIGN
Screen Shot 2020-12-03 at 11 34 35

# Acceptance Criteria
List all required acceptance criteria
- [x] Welcome guest show correctly
- [x] Welcome Screen Guest UI shares a component with Welcome Screen Register (Welcome Message).

### PR DESCRIPTION
**Pull request by develiasdaniel**
# Description
this is related issue #14 
What did you implemented/Why did you implemented this:
 - design the guest welcome screen
 - reuse components of welcome screen
 - added texts to the constants file
 
Examples:
 - What?: I have added the guest welcome screen, It included design and reuse small components such as the logo, title and description.
 - Why?: this screen will be one of the first that the user sees when they log in

 # Testing
- clone repository
- `npm install`
- `npm react native run-android` or `npm react-native run-ios`
 
 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.
  
  ## Screenshots 
- Guest Welcome Screen
GuestwelcomeScreen

### CODE REVIEW

**Review by tovl1 — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by Ocosmee — DISMISSED** [review]

**Review by JordiGN — DISMISSED** [review]
Great work, clean as the other one! cool

File: tribomobile/components/modals/ModalInfoStore.js [hunk 1]
```diff
@@ -64,16 +65,16 @@ function ModalInfoStore() {
                   information={'Tienda de abarrotes, frutas y verduras'}
                   style={styles.descriptionLeft}
                 />
-              </View>
+              </View>
```
Comment (LeyaimMagma) [inline tribomobile/components/modals/ModalInfoStore.js]: Remove the blanks paces

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Comment by LeyaimMagma** [comment]
Resolve Conflicting files

File: tribomobile/components/modals/ModalInfoStore.js (hunk 1 above)
Comment (develiasdaniel) [reply tribomobile/components/modals/ModalInfoStore.js]: Done

**Review by JordiGN — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Comment by Ing-Heber** [comment]
Well done Dany!!

**Review by JordiGN — DISMISSED** [review]

**Review by Ocosmee — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by tovl1 — DISMISSED** [review]

**Review by JordiGN — APPROVED** [review]

**Review by Ing-Heber — APPROVED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**